Networking/Networking/Remote/MetaDataRemote.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Interface for remote requests to Product and Orders meta data.
+///
+public protocol MetaDataRemoteProtocol {
+    func updateMetaData(for siteID: Int64, for parentID: Int64, type: MetaDataType, metadata: [[String: Any]]) async throws -> [MetaData]
+}
+
+public final class MetaDataRemote: Remote, MetaDataRemoteProtocol {
+
+    // MARK: - MetaData
+
+    /// Updates metadata for a specific item.
+    ///
+    /// - Parameters:
+    ///     - site: Site for which we'll update the metadata.
+    ///     - request: The request containing metadata to be updated.
+    ///     - type: The type of item (order or product) for which we'll update the metadata.
+    /// - Returns: An array of updated MetaData.
+    ///
+    public func updateMetaData(for siteID: Int64, for parentID: Int64, type: MetaDataType, metadata: [[String: Any]]) async throws -> [MetaData] {
+
+        let parameters: [String: Any] = [ParameterKey.metaData: metadata, ParameterKey.fields: "meta_data"]
```

Comment:
nit: Can we move "meta_data" to a constant enum? Or can we reuse the `ParameterKey.metaData`?

Reply:
7c567d566ff825f999e81bf9dc88622aa477b72c